internal/kafka/internal/api/private/api/openapi.yaml

```diff
@@ -20,6 +20,11 @@ tags:
 paths:
   /api/kafkas_mgmt/v1/obs-proxy-verification/{cluster_external_id}:
     post:
+      description: |-
+        this endpoint is used to check whether a token sent from a data plane cluster matches the data plane cluster details
```

Comment:
Is this endpoint for both enterprise and non-enterprise clusters?

Reply:
right now its only for enterprise. @pb82 - will this only be used for enterprise or is there a possibility that it might be used for non-enterprise clusters in the future as well?

Reply:
It's going to be used across the whole fleet - enterprise and non enterprise.

Reply:
In that case the description looks good to me.
I approved the PR under that assumption.